concepts/05 Widgets/Drawer/Getting Started with Navigation Drawer/Getting Started with Navigation Drawer.md

```diff
@@ -12,791 +12,6 @@ If the templates are unsuitable or you use jQuery, follow the instructions in th
 
```

Comment:
In line 11 above:
`The **Drawer** is opened and closed via a button on a toolbar.`

I'd like to rewrite this without using "via". Can we say the following?

`A toolbar button is used to open and close the **Drawer**.`